src/everest/simulator/everest_to_ert.py

```diff
@@ -324,9 +324,9 @@ def _is_dir_all_geo(source, ever_config: EverestConfig):
     - some are non-existing, raises an AssertionError
     """
     realizations = ever_config.model.realizations
-    if not realizations:
+    if not realizations:  # and "<GEO_ID>" in source:
         msg = "Expected realizations when analysing data installation source"
-        raise AssertionError(msg)
+        raise ValueError(msg)
```

Comment:
I guess no tests expect this to still fail with an AssertionError?

Reply:
No, there was no test for the previous `AssersionError`.